compose/reference/envvars.md

```diff
@@ -4,139 +4,165 @@ keywords: fig, composition, compose, docker, orchestration, cli, reference
+**Important**
+
+> **This setting is only intended as a workaround and it is not officially supported.**
+>
+> Set and use this variable only as a last resort workaround when you need
+> to *temporarily* run Docker with a mismatch between the client and server version. 
+> For example, when you can upgrade the client but need to wait to upgrade the server.
+>
+> This mismatch between server and client prevents some features from working properly. 
+> What features fail when using this configuration depends on the
+> Docker client and server versions in question. 
+>
+> It is highly recommended that you upgrade client and server and remove this setting as 
+> as soon as possible. Also, perform these actions to check if any problems you might 
+> be having are resolved before notifying support.
 
 ## DOCKER\_HOST
 
-Sets the URL of the `docker` daemon. As with the Docker client, defaults to `unix:///var/run/docker.sock`.
+Sets the URL of the Docker daemon. 
+* **Defaults to:** `unix:///var/run/docker.sock`(same as with the Docker client).
 
 ## DOCKER\_TLS\_VERIFY
 
 When set to anything other than an empty string, enables TLS communication with
-the `docker` daemon.
+the Docker daemon.
 
 ## DOCKER\_CERT\_PATH
 
-Configures the path to the `ca.pem`, `cert.pem`, and `key.pem` files used for TLS verification. Defaults to `~/.docker`.
+Configures the path to the `ca.pem`, `cert.pem`, and `key.pem` files used for TLS verification.
+
+* **Defaults to:** `~/.docker`.
 
 ## COMPOSE\_HTTP\_TIMEOUT
 
-Configures the time (in seconds) a request to the Docker daemon is allowed to hang before Compose considers
-it failed. Defaults to 60 seconds.
+Defines the timeout period (in seconds) for requests to the Docker daemon after which Compose considers the requests have failed.
+
+* **Defaults to:** 60 seconds.
 
 ## COMPOSE\_TLS\_VERSION
 
-Configure which TLS version is used for TLS communication with the `docker`
-daemon. Defaults to `TLSv1`.
-Supported values are: `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+Defines which TLS version is used for TLS communication with the Docker daemon. 
+
+* **Supported values:** `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+* **Defaults to:** `TLSv1`.
 
 ## COMPOSE\_CONVERT\_WINDOWS\_PATHS
 
-Enable path conversion from Windows-style to Unix-style in volume definitions.
-Users of Docker Machine on Windows should always set this. Defaults to `0`.
-Supported values: `true` or `1` to enable, `false` or `0` to disable.
+When enabled, Compose performs path conversion from Windows-style to Unix-style in volume definitions.
+
+* **Supported values:** 
+    * `true` or `1`, to enable, 
+    * `false` or `0`, to disable.
+
+* **Defaults to:** `0`
 
 ## COMPOSE\_PATH\_SEPARATOR
 
-If set, the value of the `COMPOSE_FILE` environment variable is separated
-using this character as path separator.
+Specifies a different path separator for items listed in `COMPOSE_FILE`. 
+
+Default path separator in Linux and macOS the path separator is `:`, on Windows it is `;`.
```

Comment:
This block diverges from the other  `* **Defaults to:**` blocks

Reply:
Updates.